lesson19/js/script.js

```diff
@@ -0,0 +1,172 @@
+"use strict";
+
+// Task 1 -----------------------------------------------------------
+
+const randomObj = {
+  color: 'red',
+  width: 40,
+};
+
+for (let key in randomObj) {
+  delete randomObj[key];
+}
+
+console.log(randomObj);
+
+// Task 2 -----------------------------------------------------------
+
+const randomObj1 = {
+  color: 'black',
+  width: 60,
+};
+
+function keyInObj(key, obj) {
+  return key in obj ? true : false;
+}
+
+console.log(keyInObj('color', randomObj1));
+
+// Task 3 -----------------------------------------------------------
+
+const student = {
+  name: 'John',
+  age: 19,
+  isHappy: true
+};
+
+for (let key in student) {
+  console.log(key);
+  console.log(student[key]);
+}
+
+// Task 4 -----------------------------------------------------------
+
+const colors = {
+  'ru pum pu ru rum': {
+      red: 'красный',
+      green: 'зеленый',
+      blue: 'синий'
+  },
+};
+
+const {red, blue} = colors['ru pum pu ru rum'];
+console.log(red, blue);
+
+// Task 5 -----------------------------------------------------------
+
+const salaries = {
+  andrey: 500,
+  sveta: 413,
+  anton: 987,
+  lesha: 664,
+  alexandra: 199,
+};
+
+const humans = Object.keys(salaries);
+let sumSalaries = 0;
+
+for (let key in salaries) {
+  sumSalaries += salaries[key];
+}
+
+const averageSalary = sumSalaries / humans.length;
+console.log(averageSalary);
+
+// Task 6 -----------------------------------------------------------
+
+const usersDB = {
+  user01 : {}
+};
+
+usersDB.user01.login = prompt('Enter your login:', '');
+usersDB.user01.password = prompt('Enter your password:', '');
+
+while (true) {
+  let ask1 = prompt('Confirm login:', '');
+  let ask2 = prompt('Confirm password:', '');
+  if (  ask1 === '' || ask2 === '' || ask1 === null || ask2 === null) {
```

Comment:
```suggestion
  if (ask1 === '' || ask2 === '' || ask1 === null || ask2 === null) {
```
Я уже писал, что за такое делают выговор?